### DISCUSSION TITLE
implement function demo/images_demo.py

### DISCUSSION THREAD

**Discussion by alaa-shubbak** [open-mmlab/mmdetection#9315] [Q&A] [answered]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### 💬 Describe the reimplementation questions

There is no error , i just want to know how can i use function `demo/images_demo.py` or any  other functions in this repository to test my images of exited model and weight for only one and certain category , not for the 80 categories within coco dataset. 

I try the function on my images , need to have only one category presented but it gave me all objects within the image that have been included in the 80 items in coco dataset. 

how can i remove the other categories ,without pretrained or do transfer learning for my models? 

thanks . 

### Environment

<img width="1122" alt="image" src="https://user-images.githubusercontent.com/10245810/201246224-0c689540-7451-49f9-9f11-0e8f452b426a.png">

**Comment by sanbuphy** [comment, marked as answer]
You can write a script to obtain the image address by passing through the annotations.json.
In the annotations you can find the category_id and address.
if the category_id is you want, just recording address in new variable and detect it .
